dbt_project.yml

```diff
@@ -1,7 +1,8 @@
 
 name: 'linkedin_source'
-version: '1.0.0'
+version: '0.2.0'
 config-version: 2
+require-dbt-version: [">=0.18.0", "<0.19.0"]
 
 source-paths: ["models"]
```

Comment:
Can you please remove this block of default code? We should remove any default configs from `dbt_project.yml` files that are not required, in an effort to keep things as clean and simple as possible.